09LessonNine/in-class-project/service.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Lanscaping and Gardening</title>
+    <link rel="stylesheet" href="css/css-reset.css">
+    <link rel="stylesheet" href="css/class2.css">
+</head>
+
+<body>
+    <main>
+            <section id="mainbox">
+                <div id="leftbox" class="clearfix">
+                    <div id="small-growing">
+                        <h2>Growing</h2>
+                        <p>We do it cause its our passion</p>
+                    </div>
+                    <div id="plant-care">
+                        <h2>Plant Care</h2>
+                        <p>We love what we do</p>
+                    </div>
+                </div>
+
+                <div id="rightbox" class="clearfix">
+                    <h2>Landscaping & Gardening</h2>
```

Comment:
Use `&amp;` instead of `&` so your code doesn't have any problems when it's being parsed